api/renter.go

```diff
@@ -275,8 +287,24 @@ func (api *API) renterContractsHandler(w http.ResponseWriter, _ *http.Request, _
 
 // renterDownloadsHandler handles the API call to request the download queue.
 func (api *API) renterDownloadsHandler(w http.ResponseWriter, _ *http.Request, _ httprouter.Params) {
+	dlq := api.renter.DownloadQueue()
+
+	// Translate []modules.DownloadInfo to []api.DownloadInfo.
+	downloads := make([]DownloadInfo, len(dlq))
+	for i := range dlq {
+		d := dlq[len(dlq)-i-1]
```

Comment:
is the intent here to reverse? How about instead, keep the order of `DownloadQueue`, and then use `sort.Slice` to order `downloads` as desired?